docs/fundamentals/full-stack-development.md

```diff
@@ -1,10 +1,10 @@
-# Building Full-Stack Applications using Angular and NestJS
+# Building Full-Stack Applications using Angular and Nest
 
-In this guide we will build a full-stack application using Angular and NestJS.
+In this guide you will build a full-stack application using Angular and Nest.
```

Comment:
I like the change to `you`. It gives a better sense of goal and achievement to the user